hail/python/hail/utils/java.py

```diff
@@ -82,8 +82,8 @@ def fs():
         return Env.backend().fs
 
     @staticmethod
-    def sql_context():
```

Comment:
Is this "public API"? We can't remove/change this without breaking backwards compatibility.

Reply:
no, this is on Env.